dvc/remote/base.py

```
@@ -257,8 +239,7 @@ def _download_dir(
                 )
             )
             with ThreadPoolExecutor(max_workers=self.remote.JOBS) as executor:
-                futures = executor.map(download_files, from_infos, to_infos)
-                return sum(futures)
+                executor.map(download_files, from_infos, to_infos)
```

Oops, swallowing exceptions. Will fix.